### ISSUE TITLE
ship src as esm

### ISSUE THREAD

**Issue by bennypowers** [dmurvihill/firebase-mock#52]
Shipping `src` as esm would be extremely useful. It would obviate the need for a `browser` bundle, and would be interoperable.

A `cjs` build could be shipped along side that uses `require`

If you're interested, I can issue a PR

**Comment by dmurvihill** [comment]
Yes please, ESM is awesome and it would be great to add support for it and eventually remove the CommonJS browser bundle. Thanks!

We will have to have a community discussion about our migration strategy.
